### PR TITLE
Improve download policy value selection

### PR DESCRIPTION
**Pull request by sbernhard**
Co-authored-by: Ewoud Kohl van Wijngaarden <ewoud@kohlvanwijngaarden.nl>

### CODE REVIEW

**Comment by sbernhard** [comment]
Thanks at @ekohl for your suggestions!

File: app/models/scc_account.rb [hunk 1]
```diff
@@ -287,9 +287,13 @@ def invalidate_subscription_status(items_to_invalidate, save_record:)
   end
 
   def self.download_policy_selection_values
-    names = { 'on_demand' => _('On Demand'), 'immediate' => _('Immediate') }
+    names = {
+      ::Katello::RootRepository::DOWNLOAD_ON_DEMAND => _('On Demand'),
+      ::Katello::RootRepository::DOWNLOAD_IMMEDIATE => _('Immediate')
```
Comment (nadjaheitmann) [inline app/models/scc_account.rb:292]: Shouldn't it be like this: 
```suggestion
      'on_demand' => ::Katello::RootRepository::DOWNLOAD_ON_DEMAND,
      'immediate' => ::Katello::RootRepository::DOWNLOAD_IMMEDIATE
```

**Review by ekohl — COMMENTED** [review]
I'd also consider having this mapping in Katello itself. For example, there already is https://github.com/Katello/katello/blob/53b2a5677c5eb90d115b8eb74bdd34c4f2674b08/app/helpers/katello/concerns/smart_proxy_helper_extensions.rb#L28-L49

**Review by m-bucher — APPROVED** [review]
Looks good to me.

**Pull request merged**